tests/integration/components/imgix-image-wrapped-test.js

```diff
@@ -0,0 +1,120 @@
+import { moduleForComponent, test } from 'ember-qunit';
+import hbs from 'htmlbars-inline-precompile';
+
+moduleForComponent(
+  'imgix-image-wrapped',
+  'Integration | Component | imgix image wrapped',
+  {
+    integration: true
+  }
+);
+
+test('it renders', function(assert) {
+  this.render(hbs`{{imgix-image-wrapped path="/users/1.png"}}`);
+  assert.ok(this.$());
+});
+
+test('it renders event more better', function(assert) {
```

Comment:
I don't think having duplicate tests in this file is particularly useful to us. Since this component is just a wrapper around `imgix-image`, we should just test that the component does indeed wrap correctly and move on.

Reply:
The idea here was to ensure that wrapped component passes all the same tests as the main component as it doesn't share any code with the main component. Therefore I don't think it works to check that it wraps correctly, as we aren't testing the underlying implementation.

The wrapped component is the legacy component which is kept around to make migration easier.

Reply:
Ah my bad, I thought it was literally just a wrapper. Carry on.